### ISSUE TITLE
Community: Start a circle - no indication that clicking "create" is performing an action

### ISSUE THREAD

**Issue by chayaml** [ji-devs/ji-cloud#3245]
The create button is gray when you start a circle and fill in all the fields, but turns red once an image is uploaded. After clicking "create", the button turns gray again - should it turn green to indicate progress? Currently, it appears as if nothing is happening, since the new circle takes a few moments to load.

![Screen Shot 2022-10-03 at 1 11 39 PM](https://user-images.githubusercontent.com/78186896/193556272-cd40437d-5874-48a2-ab06-6b62c1031eb0.png)

**Comment by corinnewo** [comment]
You are right that we need some reaction, but not to turn the button green, as we do not have that behaviour anywhere. This would be a perfect place for the loader, just as an indication that something has been activated and is happening.
@GaliShapira What do you think?

**Comment by GaliShapira** [comment]
I just tried it. 
 You are right, it doesnt make sence. The idea is to move back to the groups list and see your created group on top.
 We need the loader and if possible to check why moving to the next screen takes so long.

**Comment by corinnewo** [comment]
Please add in the loader to show that something is happening. Still takes a while, and even with new community section, this is needed.
